From 584d4bcae652f4306b2dbe2108ce2f195d56d6f7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bastien=20ROUCARI=C3=88S?= <roucaries.bastien@gmail.com>
Date: Mon, 21 Aug 2017 23:10:30 +0200
Subject: Remove depends unicode-length

Not totally correct but it will change something only for 14 characters.

Will revise when unicode-length will be packaged

Forwarded: no
---
 lib/reporters/classic.js | 3 ++-
 package.json             | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/lib/reporters/classic.js b/lib/reporters/classic.js
index 2545227..d87fec3 100644
--- a/lib/reporters/classic.js
+++ b/lib/reporters/classic.js
@@ -9,7 +9,8 @@ var Base = require('./base')
   , ms = require('../ms.js')
   , diff = require('diff')
   , utils = require('../utils.js')
-  , uclen = require('unicode-length').get
+  , stripAnsi = require('strip-ansi')
+  , uclen = function(str) { return stripAnsi(str).length }
   , colorSupport = require('supports-color')
 
 function repeat (n, c) {
diff --git a/package.json b/package.json
index d5d9dea..be2a296 100644
--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "glob": "^7.0.5",
     "js-yaml": "^3.3.1",
     "tap-parser": "^5.1.0",
-    "unicode-length": "^1.0.0"
+    "strip-ansi" : "^3.0.0"
   },
   "devDependencies": {
     "tap": "^6.2.0"
